cvd/allocd/alloc_utils.cpp

```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-#include "host/libs/allocd/alloc_utils.h"
+#include "alloc_utils.h"
```

Comment:
nit: should this now be `allocd/`?

Reply:
The stuff under `cuttlefish/host/libs/<module>/<header>` are included as `"host/libs/<module>/<header>` so I thought the most symmetric way to do it here is to exclude the top level `allocd` directory. I don't have a preference either way though.